README.md

```
@@ -308,6 +308,83 @@ export default route.page({
+export default route.page({
+  // Enumerate all paths — called once at startup, not per-request.
+  staticParams: () => getPosts().map((post) => ({ slug: post.slug })),
+
+  loader: ({ params }) => ({ post: getPost(params.slug) }),
+
+  component: ({ post }) => <Article post={post} />,
+});
+```
+
+On production start you will see:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)